### PR TITLE
Support smallint, tinyint, date type of Cassandra

### PR DESCRIPTION
**Pull request by SandishKumarHN**
Support smallint, tinyint, date type of Cassandra

```
== RELEASE NOTES ==
Cassandra Change
* Add `SMALLINT`, `TINYINT`, and `DATE` type support to Cassandra connector.
```

### CODE REVIEW

**Comment by dborkar** [comment]
@highker or @wenleix , Can you please help review? Thx.

File: presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraType.java [hunk 1]
```diff
@@ -127,6 +134,8 @@ public static CassandraType getCassandraType(DataType.Name name)
                 return COUNTER;
             case CUSTOM:
                 return CUSTOM;
+            case DATE:
```
Comment (fgwang7w) [inline presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraType.java:137]: better to document these types support in cassandra.rst

**Comment by fgwang7w** [comment]
LGTM. I'd defer to @highker if there's any more comments. 
@SandishKumarHN There is an integration test failure, please double check if you need to do rebasing. Thanks!

**Comment by SandishKumarHN** [comment]
> LGTM. I'd defer to @highker if there's any more comments.
> @SandishKumarHN There is an integration test failure, please double check if you need to do rebasing. Thanks!

@fgwang7w the integration test failure is not related to PR.

**Review by highker — APPROVED** [review]
Could you sign CLA?

**Comment by highker** [comment]
and squash two commits into one?

**Comment by SandishKumarHN** [comment]
> and squash two commits into one?

@highker done, and CI failure is not related to the PR.

**Comment by ajaygeorge** [comment]
Unfortunately this has resurfaced the joda libraries which were taken out in https://github.com/prestodb/presto/pull/14046 :(

**Pull request merged**